### DISCUSSION TITLE
Setup development environment on debian

### DISCUSSION THREAD

**Discussion by BbqGamer** [FossifyOrg/General-Discussion#235] [Q&A]
Hello
I love fossify apps and wanted to contribute by introducing some changes. I know how to code but I am completely new to kotlin and android development in general. I had a lot of trouble in setting up the dev environment, i am working on Debian and wasn't able to build the app to generate the APK file. Has anybody thought about providing some kind of dev guide, about what packages are needed to jump start the process?

**Comment by Aga-C** [comment]
There's no guide because the only tool you need is [Android Studio](https://developer.android.com/studio/install?hl=pl#linux) which is available for all systems.
